src/one_dragon_qt/widgets/color_tip.py

```
@@ -0,0 +1,190 @@
+            if not self._validate_color_info(color_info):
+                continue
+
+            color_widget = ColorInfoWidget(color_info)
+            layout.addWidget(color_widget)
+            if idx < len(self.color_infos) - 1:
+                separator = QWidget()
+                separator.setFixedSize(1, 170)
+                separator.setStyleSheet("background-color: #c0c0c0;")
+                layout.addWidget(separator, alignment=Qt.AlignmentFlag.AlignVCenter)
+
+        # 如果没有有效的颜色信息，显示错误消息
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)